### ISSUE TITLE
302 error when creating a project

### ISSUE THREAD

**Issue by djcaesar9114** [zanran/node-redmine#19]
Hello,
When I create a project, I receive this error:

`{"ErrorCode":302,"Message":"Moved Temporarily","Detail":"<html>\r\n<head><title>302 Found</title></head>\r\n<body bgcolor=\"white\">\r\n<center><h1>302 Found</h1></center>\r\n<hr><center>nginx/1.6.2</center>\r\n</body>\r\n</html>\r\n"}`

Here is my syntax:

```
redmine.create_project({project: {name: nom_projet, identifier: 'lol', description: 'Testouille'}}, function(err, data) {
    if (err) {
      console.log(err);
      resultat = 'ko';
    }
    else {
      resultat = 'ok';
    }
    req.nom_projet = nom_projet;
    req.resultat = resultat;
    return next();
  });
```

When I'm using CURL, everything is OK with project creation API.

Did I write something wrong?
Thanks in advance.

**Comment by djcaesar9114** [comment]
OK, my bad, I'm using nginx with a redirection from HTTP to HTTPS:
```
  server_name xxx.fr;
  rewrite ^ https://xxx.fr$request_uri?;
```
I configured node-redmine to make calls to http, and I got a 302. Now I configured node-redmine to call https, and everything works fine.
It's kind of weird, because I didn't have this problem when I wanted to fetch issues.
I don't know if that's a node-redmine issue or a Node misunderstanding from me. I think the latter, and if somebody could tell me how I can make my code work without having to precise HTTPS in my app configuration, it would be great.
